index.html

```diff
@@ -59,7 +59,7 @@
 						<h2><span>User Group</span></h2>
 						<img src="images/logo-white.png">
 						<h3>Meetups so far: <span>9</span></h3>
-						<h3>Pythonista in Jaipur: <span>1250+</span></h3>
+						<h3>Pythonista in Jaipur: <span>1300+</span></h3>

```

Comment:
it would be really awesome if this number came from the membership of the Telegram group